staff/sys/ocf-kubernetes-deploy

```diff
@@ -61,20 +76,30 @@ def main():
     ).check_returncode()
 
     # Bindings passed into the kubernetes-deploy templates
-    bindings = {
-        'version': args.appversion,
-    }
 
-    # kubernetes-deploy requires that this environmental variable be set, and passes
-    # it to templates as the `current_sha` variable. We don't ever use it.
-    os.environ['REVISION'] = 'unused'
+    # Created with 600 perms
+    bindings_file = tempfile.NamedTemporaryFile(suffix='.yaml')
+
+    version_binding = 'version: ' + args.appversion + '\n'
```

Comment:
This is not a good way to write YAML. You should keep everything in a `bindings` dict, and write it all to a temporary JSON file at the end.